### PR TITLE
Black and pre-commit

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+jobs:
+  include:
+    - script: python -m pytest test_esss_jenkins.py
+      python: "3.6"
+      name: "Python 3.6"
+    - script: python -m pytest test_esss_jenkins.py
+      python: "3.7"
+      name: "Python 3.7"
+    - script: pre-commit run --all-files --show-diff-on-failure
+      python: "3.6"
```
Comment (gcardozo123) [inline .travis.yml:16]: Don't we need this for Python 3.7 as well?

**Review by gcardozo123 — APPROVED** [review]
Nice we work, we should do the same on `err-stash`! :)

File: .travis.yml (hunk 1 above)
Comment (nicoddemus) [reply .travis.yml:16]: You mean `pre-commit`? Hardly, 3.6 and 3.7 syntax are identical. :)

Comment (gcardozo123) [reply .travis.yml:16]: Ok :)

**Pull request merged**